library/core/src/result.rs

```diff
@@ -796,7 +818,9 @@ impl<T, E> Result<T, E> {
     /// ```
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
-    pub fn map<U, F: FnOnce(T) -> U>(self, op: F) -> Result<U, E> {
+    #[rustc_const_unstable(feature = "const_trait_impl", issue = "67792")]
+    #[rustc_allow_const_fn_unstable(const_precise_live_drops)]
```

Comment:
Aiui it's good practice to add a note when this is used

Reply:
This shouldn't be used here, the function is `rustc_const_unstable` anyway so it is unnecessary.